src/commands/destroy.ts

```diff
@@ -25,29 +25,43 @@ export default class Destroy extends DeployCommand {
     }),
   };
 
+  static sensitive = new Set();
+
+  static non_sensitive = new Set([...Object.keys({ ...this.flags })]);
+
   async run(): Promise<void> {
-    const { flags } = await this.parse(Destroy);
+    try {
```

Comment:
Is there a way to log to Sentry without wrapping each function in a try/catch block? Maybe by adding something in the base class?

Reply:
I know it's not very dry, so I'm writing a class and method decorator to get the proper context/stacktrace without the try catch blocks.  This ideally will be set on the BaseCommand class only, and once I can confirm it works the same i'll remove all of the added try/catch in this PR.

Reply:
refactored as a class decorator